packages/utils/src/PromiseUtils.ts

```diff
@@ -18,10 +18,13 @@ export class PromiseUtils {
   ): CancelablePromise<T> {
     let hasCanceled = false;
     let resolved: T | undefined;
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    let rejectFn: (val?: any) => void;
```

Comment:
You can use `unknown` here instead of `any` and get rid of the disabling eslint.